examples/mock-provider/README.md

```diff
@@ -0,0 +1,21 @@
+# Generic OAuth with mock provider
+
+This example uses [mock-oauth2-server][] to launch a standalone local OAuth2 provider and configures GenericOAuthenticator to use it.
```

Comment:
```suggestion
This example uses [mock-oauth2-server] to launch a standalone local OAuth2 provider and configures GenericOAuthenticator to use it.
```

Reply:
I always forget you can do that. Thanks!

Reply:
I don't know if this works for all markdown types, so lets ignore this suggestion maybe